Kernel/FileSystem/Ext2FileSystem.cpp

```diff
@@ -999,26 +1003,26 @@ ErrorOr<size_t> Ext2FSInode::write_bytes(off_t offset, size_t count, UserOrKerne
 
     did_modify_contents();
 
-    dbgln_if(EXT2_VERY_DEBUG, "Ext2FSInode[{}]::write_bytes(): After write, i_size={}, i_blocks={} ({} blocks in list)", identifier(), size(), m_raw_inode.i_blocks, m_block_list.size());
+    dbgln_if(EXT2_VERY_DEBUG, "Ext2FSInode[{}]::write_bytes(): After write, file_size={}, num_blocks={} ({} blocks in list)", identifier(), size(), m_raw_inode.num_blocks, m_block_list.size());
     return nwritten;
 }
 
 u8 Ext2FS::internal_file_type_to_directory_entry_type(DirectoryEntryView const& entry) const
 {
-    switch (entry.file_type) {
-    case EXT2_FT_REG_FILE:
+    switch ((Ext2::FileType)entry.file_type) {
```

Comment:
static_cast

Reply:
Why?

Reply:
Yeah, we should put that in.

Basically, C casts have very complex semantics while the semantics of all C++ casts are more or less simple and clear. We are trying to avoid C casts everywhere.